src/Umbraco.Web/PropertyEditors/RichTextPreValueController.cs

```diff
@@ -71,9 +69,10 @@ private static void EnsureInit()
                                          new RichTextEditorCommand()
                                          {
                                              IsStylePicker = isStyle,
+                                             Name = n.SelectSingleNode("./name") != null ? n.SelectSingleNode("./name").FirstChild.Value : alias,
                                              Icon = n.SelectSingleNode("./icon").FirstChild.Value,
                                              Command = n.SelectSingleNode("./tinyMceCommand").FirstChild.Value,
-                                             Alias = n.SelectSingleNode("./umbracoAlias").FirstChild.Value.ToLower(),
+                                             Alias = alias,
                                              UserInterface = n.SelectSingleNode("./tinyMceCommand").Attributes.GetNamedItem("userInterface").Value,
                                              FrontEndCommand = n.SelectSingleNode("./tinyMceCommand").Attributes.GetNamedItem("frontendCommand").Value,
                                              Value = n.SelectSingleNode("./tinyMceCommand").Attributes.GetNamedItem("value").Value,

```

Comment:
Just re-use the value for ```alias``` and check that ```<name>``` node exists in xml, otherwise you will get a configuration error that object instance is null. If the ```<name>``` node doesn't exists it fallback to ```alias```, so it shouldn't break anything if developers maybe forgot to merge changes in the config file.

It could also check for both null, empty string and whitespace, so it fallback to alias if the ```<name>``` node doesn't contain actually text.